spring-ai-model/pom.xml

```diff
@@ -60,7 +60,7 @@
 
 		<dependency>
 			<groupId>io.micrometer</groupId>
-			<artifactId>micrometer-tracing</artifactId>
+			<artifactId>micrometer-tracing-bridge-otel</artifactId>
```

Comment:
spring-ai-model should not depend on a bridge (implementation instead of the abstraction).